staging/src/k8s.io/apiserver/pkg/util/peerproxy/peerproxy.go

```diff
@@ -67,12 +67,15 @@ type Interface interface {
 	HasFinishedSync() bool
 	RunLocalDiscoveryCacheSync(stopCh <-chan struct{}) error
 	RunPeerDiscoveryCacheSync(ctx context.Context, workers int)
-	RunExcludedGVsReaper(stopCh <-chan struct{})
-	RunGVDeletionWorkers(ctx context.Context, workers int)
 	GetPeerResources() map[string][]apidiscoveryv2.APIGroupDiscovery
 	RegisterCacheInvalidationCallback(cb func())
+	// GV exclusion manager informer registration
 	RegisterCRDInformerHandlers(crdInformer cache.SharedIndexInformer, extractor GVExtractor) error
 	RegisterAPIServiceInformerHandlers(apiServiceInformer cache.SharedIndexInformer, extractor GVExtractor) error
+	// GV exclusion manager worker methods
+	RunActiveGVTracker(ctx context.Context, workers int)
```

Comment:
```suggestion
	RunPeerDiscoveryActiveGVTracker(ctx context.Context, workers int)
```

? (related to above comments)